components/Ripple/tests/snapshot/MDCStatefulRippleViewSnapshotTests.m

```diff
@@ -87,4 +87,19 @@ - (void)testDraggedRippleView {
   [self generateSnapshotAndVerifyForView:self.view];
 }
 
+- (void)testRippleHighlightedIsNotDissolvedWhenSetToYES {
+  // Given
+  [self.rippleView setRippleColor:UIColor.orangeColor forState:MDCRippleStateHighlighted];
+  self.rippleView.rippleHighlighted = YES;
+  [self.rippleView setValue:@YES forKey:@"_tapWentInsideOfBounds"];
+  [self.rippleView setValue:@YES forKey:@"_tapWentOutsideOfBounds"];
+  self.rippleView.rippleHighlighted = NO;
+  [self.rippleView setValue:@NO forKey:@"_tapWentOutsideOfBounds"];
+  self.rippleView.rippleHighlighted = YES;
+  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:2.0]];
```

Comment:
In the interest of speed a combination of what you had before (ramped up layer speed) + a low time interval here should be sufficient to ensure that the animation has taken effect.